src/Orckestra.Composer.Cart/Parameters/Order/CreateOrderDetailViewModelParam.cs

```diff
@@ -55,6 +55,8 @@ public class CreateOrderDetailViewModelParam
 
         public string OrderDetailBaseUrl { get; set; }
 
+        public List<string> EditableShipmentStatusSettings { get; set; }
```

Comment:
add to parameter the whole OrderSettings object in case the client implementation will need more properties from settings to calculate if OrderEditable